unison-src/transcripts-using-base/fix2049.output.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think I accidentally deleted this transcript in 5a7e3b49323c0beb61942527224a826b72d2d784

Reply:
Looks like you intentionally folded it into another transcript for the same fix that doesn’t require `base`: https://github.com/unisonweb/unison/commit/5a7e3b49323c0beb61942527224a826b72d2d784#diff-223e248e6ebb28c8bb9cc3875fec056c83d643571660ea2c2d12578f96631664

Reply:
Oh great -- thanks!